sdk/keyvault/azure.keyvault.keys.txt

```diff
@@ -0,0 +1,180 @@
+**** Package/Module azure.keyvault.keys****
+class KeyClient(KeyVaultClientBase, object):
+	vault_url:str
+	credential
+
+	vault_url
+
+	def __init__(self, vault_url: str, credential, transport=None, api_version=None, **kwargs): ...
+	def backup_key(self, name: str, **kwargs) -> bytes: ...
+	def create_ec_key(self, name: str, hsm: bool, curve: curve=None, key_operations: key_operations=None, enabled: bool=None, expires: datetime.datetime=None, not_before: datetime.datetime=None, tags: dict=None, **kwargs) -> ~azure.keyvault.keys.models.Key: ...
+	def create_key(self, name: str, key_type: key_type, size: int=None, key_operations: key_operations=None, enabled: bool=None, expires=None, not_before: datetime.datetime=None, tags: dict=None, curve: curve=None, **kwargs) -> ~azure.keyvault.keys.models.Key: ...
+	def create_rsa_key(self, name: str, hsm: bool, size: int=None, key_operations: key_operations=None, enabled: bool=None, expires=None, not_before: datetime.datetime=None, tags: dict=None, **kwargs) -> ~azure.keyvault.keys.models.Key: ...
+	def delete_key(self, name: str, **kwargs) -> ~azure.keyvault.keys.models.DeletedKey: ...
+	def get_cryptography_client(self, key: key, **kwargs) -> :class:`~azure.keyvault.keys.crypto.CryptographyClient`: ...
```

Comment:
Side note: this guy has no docstring. Why it's taking a "key" only and not a possible str for the key name (like some others?).

Reply:
It has a docstring. `key` is a `Union[Key, str]`. Allowable values are a `Key` instance or a key's full URI. We consider a name alone insufficient because keys are versioned and we want to encourage users to be deliberate about which version they're using in crypto operations.

Reply:
tracking with issue: #7783